backend/users/forms.py

```diff
@@ -58,3 +63,57 @@ def save(self):
         user = authenticate(email=email, password=password)
         login(self.context, user)
         return user
+
+
+class UpdateForm(ContextAwareModelForm):
+    date_birth = CharField(required=False)
+    # TODO: Why if I tell you required=FALSE, the mutation fail if I don't put
+    # Bool! (instead of 'Bool')?
+    open_to_recruiting = BooleanField(required=False)
+    open_to_newsletter = BooleanField(required=False)
+    country = CharField(required=False)
+
+    def clean(self):
+
+        if not self.context.user.is_authenticated:
+            raise ValidationError(_("Must authenticate to update User information"))
```

Comment:
Don't we have a `permission_classes` to avoid to insert this check everywhere? I think if you have a look at `SendSubmission` you should find an use of that

Reply:
fixed